### ISSUE TITLE
Cannot create an app

### ISSUE THREAD

**Issue by pluc** [soundcloud/api#127]
With the new authorization flow, for server-side application I need to create/register an App on Soundcloud so that I can have its client_id, client_secret and set its redirect_url.

That process is [apparently manual](https://soundcloud.com/you/apps/new), and the form you provide has now been disabled.

![image](https://user-images.githubusercontent.com/64719/132692756-cbedf34e-3fda-4d20-9bb2-a23c449c3aee.png)

In other words, Soundcloud is asking me to implement an authentication mechanism that it has itself disabled because it was getting too many requests to conform to the new authentication scheme it has implemented.

So how do I create an app?

**Comment by ScottBrenner** [comment]
https://github.com/soundcloud/api#can-i-request-an-api-key-for-a-new-app

**Comment by pluc** [comment]
@ScottBrenner The link you send tells me I need to get a new API key for a new app, but like I just said that registration is disabled by soundcloud itself. So thanks, but that's useless.

Can someone at soundcloud come up with an intelligent answer here? You are telling us we need to comply with a new authentication scheme but you are not letting us register for said authentication scheme, what exactly are we supposed to do to become compliant?

**Comment by pluc** [comment]
I genuinely do not understand how a company like Soundcloud can give its users an ultimatum to comply with the new authentication scheme and at the same time prevent every user from generating the apps and credentials you need to comply with the new authentication scheme. It's either completely unprofessional or technologically inept.

**Comment by mgoodfellow** [comment]
You already have an app I believe? So you just need to set a `redirect_url` - in which case: https://github.com/soundcloud/api#how-can-i-update-my-apps-redirect_uri

**Comment by pluc** [comment]
I do not want to break our existing app, so I want a new app.

**Comment by dasha-kobzeva** [comment]
Hello @pluc,

We are very sorry that the recent changes created frustration for you. We are here to help, but we won't be tolerating disrespectful speech. Please make sure you comply with our Code of Conduct: https://github.com/soundcloud/api#code-of-conduct

Regarding your request, the security changes are meant for already existing applications. The registration of new apps is closed at the moment.

We have not given any ultimatums. We are politely asking to update the existing systems, so both your app and our system can be more secured.

Thank you

**Comment by ichiniro** [comment]
Will it take a long time to accept new applications? I have to create an app for the university and I would like it to be a SoundCloud client.

**Comment by dasha-kobzeva** [comment]
> Will it take a long time to accept new applications? I have to create an app for the university and I would like it to be a SoundCloud client.

@ichiniro, I cannot provide you any estimate at this time.

**Comment by pluc** [comment]
@ichiniro  If you don't already have an app you efficiently cannot use the API, and they are not letting users create apps themselves. Even to edit existing apps you have to contact support, which is beyond unbelievable for a technology company. I suggest you find a platform with a decent developer experience, which is absolutely not this. If this is the state of things at Soundcloud, then it must be dying and I would migrate away.
